API/Extensions/DbInitializerExtension.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.EntityFrameworkCore;
+using VL.Shared.Data;
+using VL.Shared.Interfaces;
+
+namespace API.Extensions;
+
+internal static class DbInitializerExtension
+{
+    public static async Task MigrateDatabasesAsync(this WebApplication app)
+    {
+        var scope = app.Services.CreateScope();
+        var applicationDbContext = scope.ServiceProvider.GetService<ApplicationDbContext>();
+        var services = scope.ServiceProvider;
+        var migrateService = services.GetRequiredService<IMigrateService>();
+        var result = await migrateService.CheckRoles();
```

Comment:
You are checking if you have roles and if not, migrating the database - you should always call migrate async on start however